holidays/countries/suriname.py

```
@@ -0,0 +1,257 @@
+            dt = self._add_holiday_aug_9(tr("Dag der Inheemsen"))
+            if self._year > 2024:
+                self._add_observed(dt)
+
+        if self._year >= 2012:
+            # Day of the Maroons.
+            dt = self._add_holiday_oct_10(tr("Dag der Marrons"))
+            if self._year > 2024:
+                self._add_observed(dt)
+
+        if self._year >= 1976:
+            name = (
```

Ruff (0.8.2)

Local variable `name` is assigned to but never used

Remove assignment to unused variable `name`

(F841)